paper-dialog-common.css

```diff
@@ -21,26 +21,25 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   @apply(--paper-dialog);
 }
 
-:host > ::content > * {
+:host > ::content > *:not(.header):not(.buttons) {
```

Comment:
@sorvell says that this is bad news bears (it's really slow, not to mention very deprecated in shadow DOM v1), so I don't think this is a good change. :(